src/python/pants/backend/experimental/terraform/lint/trivy/register.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2024 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+from pants.backend.terraform.lint.trivy.rules import rules as trivy_terraform_rules
+from pants.backend.tools.trivy.rules import rules as trivy_rules
+
+
+def rules():
+    return (
+        # TODO: Register Terraform rules
```

Comment:
nit: shouldn't this TODO be addressed?

Reply:
sorry, I thought I had done all the TODOs D: